Source/ScalingHeaderScrollView.swift

```diff
@@ -59,6 +59,9 @@ public struct ScalingHeaderScrollView<Header: View, Content: View>: View {
     /// Allow force snap to closest position after lifting the finger, i.e. forbid to be left in unfinished state
     private var allowsHeaderSnapFlag: Bool = false
     
+    /// Shows or hidde the indictor for the scrollview
```

Comment:
There are some typos, would you please use this phrase?
Shows or hides the indicator for the scrollView

Reply:
done